parts/k8s/kubernetesconfigs.sh

```diff
@@ -408,7 +408,7 @@ configAddons() {
 }
 
 configGPUDrivers() {
-    retrycmd_if_failure 30 1 120 sh $GPU_DEST/nvidia-drivers-$GPU_DV --silent --accept-license --no-drm --dkms --utility-prefix="${GPU_DEST}" --opengl-prefix="${GPU_DEST}" || exit $ERR_GPU_DRIVERS_START_FAIL
+    retrycmd_if_failure 30 1 1200 sh $GPU_DEST/nvidia-drivers-$GPU_DV --silent --accept-license --no-drm --dkms --utility-prefix="${GPU_DEST}" --opengl-prefix="${GPU_DEST}" || exit $ERR_GPU_DRIVERS_START_FAIL
```

Comment:
this is too aggressive and will cause CSE timeouts (+ 90 minutes) on the ARM deployment if it actually fails every time.

Reply:
Good point, what do you think about my latest commit that reduces the retry count to 3?